lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -136,9 +136,9 @@ const std::string &SPIRVToLLVMDbgTran::getString(const SPIRVId Id) {
 }
 
 const std::string
-SPIRVToLLVMDbgTran::getStringContinued(const SPIRVId Id,
-                                       SPIRVExtInst *DebugInst) {
-  if (getDbgInst<SPIRVDebug::DebugInfoNone>(Id))
+SPIRVToLLVMDbgTran::getStringSourceContinued(const SPIRVId Id,
+                                             SPIRVExtInst *DebugInst) {
+  if (!isValidId(Id) || getDbgInst<SPIRVDebug::DebugInfoNone>(Id))
```

Comment:
That looks fishy, could you please explain what you have observed during testing?

Reply:
This check was added because of code on line 1509 where I explicitly set Id to invalid value to mark that we don't have Source operand in some cases.